### PR TITLE
Update on usage examples

### PR DESCRIPTION
**Pull request by tongokongo**
Added some usage examples, had some problems formatting in rst format. 
Not sure if the project is still supported for python 2.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,3 +22,53 @@ BIP Paper
...
+
+.. code-block:: python
+
+   seed = mnemo.to_seed(words, passphrase="") 
+
+Given the word list, calculate original entropy:
+
+.. code-block:: python
+
+   entropy = mnem.to_entropy(words)
```
Comment (matejcik) [inline README.rst:74]: typo `mnem`

**Comment by matejcik** [comment]
we have recently dropped Python 2 support. not sure how that is relevant to the examples?

**Comment by tongokongo** [comment]
The only part of the usage examples possibly dependent on the python version is `pip install`. When python3 is a default python version, it will work, if not the pip3 command has to be explicitly invoked.

**Comment by matejcik** [comment]
merged

**Pull request closed without merging**